### DISCUSSION TITLE
Files and folders are not cleaned after setup

### DISCUSSION THREAD

**Discussion by ryan-conway** [nimblehq/android-templates#284] [RFCs]
After running the setup scripts, the project folder is created inside the template root folder and all other files are left as is

For example, we run `kscript new_project.kts package-name=co.nimblehq.suvkmmsurveys.android app-name=Surveys`. 
It may be unclear if `Surveys` or `template` is the right folder 🤔

<img width="489" alt="Screen Shot 2022-08-23 at 10 32 57" src="https://user-images.githubusercontent.com/19943832/186063648-076b32e0-37b5-4395-a557-7e255cf9cd2e.png">

[Original issue](https://github.com/nimblehq/android-templates/issues/283)

**Comment by lydiasama** [comment]
IMO, the delete flag might be the best option but I found that there could be a chance we get errors when generating the project and might need to go back to the script and re-run it again. 😢

**Comment by toby-thanathip** [comment]
I voted for "Add an argument to specify output for folder location" ☑️ 

This option provides the most flexibility 🚀

**Reply by kaungkhantsoe** [reply]
I was thinking about `Add an argument to specify output for folder location` option lately but didn't know we had this discussion. 👍

**Comment by hoangnguyen92dn** [comment]
Why don't we add both? 🤔 So developers have more option instead 😄 

```
- Add a feature flag to delete the template repo (-d == `delete the template after generating`)
- Add an argument to specify output folder location
```

**Comment by chornerman** [comment]
IMO, I think the current approach already makes sense to me, so I voted "Do nothing" 🙏🏼

**Comment by ryan-conway** [comment]
Issue created: #436
